lib/rubocop/cop/style/unneeded_percent_q.rb

```diff
@@ -14,6 +14,7 @@ def on_dstr(node)
         end
 
         def on_str(node)
+          return unless standalone_string?(node)
```

Comment:
I'll think about this a bit more - at the very least `standalone_string?` should be `string_literal?` and there should probably be some explanation about why this check is needed. The cop could use a better class comment in general. Probably using the same `check` method for both cases wasn't a great idea either.